roles/openssl_cert_gen/templates/xdmod.conf

```diff
@@ -0,0 +1,50 @@
+<IfModule !mod_ssl.c>
+    LoadModule ssl_module modules/mod_ssl.so
+</IfModule>
+
+Listen 443
+
+<VirtualHost *:80>
+    ServerName {{ ServerName }}
+    Redirect permanent / https://{{ ServerName }}/
+</VirtualHost>
+
+<IfModule mod_ssl.c>
```

Comment:
Can remove this check here as well. In my latest deploy I removed it and it still works

Reply:
Yeah, I think we can remove this check.

Reply:
done